cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/AuthorizationBootstrapperTest.java

```diff
@@ -197,7 +197,8 @@ public Boolean call() throws Exception {
     Assert.assertNotNull(systemDataset);
     // as part of bootstrapping, admin users were also granted admin privileges on the CDAP instance, so they can
     // create namespaces
-    SecurityRequestContext.setUserId(ADMIN_USER.getName());
+    SecurityRequestContext.setUserId(ADMIN_GROUP.getName());
```

Comment:
Why set group to userId ? I think you need a mock UserGroupMappingService for the unit test